lavamoat/browserify/beta/policy.json

```diff
@@ -5453,6 +5464,13 @@
         "@ensdomains/content-hash>multicodec>uint8arrays>multiformats": true
       }
     },
+    "@metamask/keyring-controller>ulid": {
+      "globals": {
+        "console.error": true,
+        "crypto": true,
+        "define": true
+      }
+    },
```

Comment:
For future reviewers: Now required by the keyring ID generation (required by the multi-SRP feature)